controllers/windowsmachine_controller.go

```diff
@@ -247,6 +247,11 @@ func (r *WindowsMachineReconciler) Reconcile(ctx context.Context, request ctrl.R
 		err := r.client.Get(ctx, kubeTypes.NamespacedName{Namespace: machine.Status.NodeRef.Namespace,
 			Name: machine.Status.NodeRef.Name}, node)
 		if err != nil {
+			// Do not requeue if associated node cannot be found (i.e. deleted) for a running machine
+			if k8sapierrors.IsNotFound(err) {
+				log.Info("could not find node associated with machine", "name", machine.GetName())
```

Comment:
nit : should we add something like no-op to the log indicating we are not taking any action, similar to what machine-api does?
/lgtm

Reply:
I agree that this log message could use some re-assurance that this is not an error on WMCO's side, something like:
`the Node associated with this machine does not exist, no action taken`? 
Or however you want to word it

Reply:
Good points, taken care of